docs/pages/workflow/prebuild.mdx

```diff
@@ -24,7 +24,16 @@ We highly recommend using prebuild for the reasons listed in the [pitch](#pitch)
 
 ### Usage with EAS Build
 
-You can configure EAS Build to run `npx expo prebuild` before building by [using the managed preset](/build-reference/ios-builds/).
+If your project does not contain **ios** and **android** folders, then EAS Build will run Prebuild to generate your native code just before compilation. This is the default for any project started from `npx create-expo-app`.
```

Comment:
```suggestion
If your project does not contain **android** and **ios** directories, EAS Build will run Prebuild to generate these native directories before compilation. This is the default for any project created from `npx create-expo-app`.
```

- [Platform order](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-android-ios-and-web) as per writing style guide
- Directory over folder for consistency